### ISSUE TITLE
Maybe each user should see only their own images?

### ISSUE THREAD

**Issue by csawtelle** [gerardramosm89/mscsblog#53]
When you go to the dashboard you see every image on the server, might be handy if it was filtered by active user?

**Comment by gerardramosm89** [comment]
yeah! I was thinking the same for this issue, definitely doable, putting this on the list for sure

**Comment by csawtelle** [comment]
https://github.com/csawtelle/mlhq.io/issues/276
